Sources/PackageDescription/LanguageStandardSettings.swift

```diff
@@ -10,161 +10,160 @@
-/// To learn more, read the [Clang Compiler User's Manual][1].
-///
-/// [1]: <https://clang.llvm.org/docs/UsersManual.html#differences-between-various-standard-modes>
+/// The supported C language standard to use for compiling C sources in the
+/// package.
 public enum CLanguageStandard: String, Encodable {
 
-    /// ISO C 1990.
+    /// The identifier for the C89 language standard.
     case c89
 
-    /// ISO C 1990.
+    /// The identifier for the C90 language standard.
     case c90
 
-    /// ISO C 1999.
+    /// The identifier for the C99 language standard.
     case c99
 
-    /// ISO C 2011.
+    /// The identifier for the C11 language standard.
     case c11
 
-    /// ISO C 2017.
+    /// The identifier for the C17 language stadard.
     @available(_PackageDescription, introduced: 5.4)
     case c17
 
-    /// ISO C 2017.
+    /// The identifier for the C18 language standard.
     @available(_PackageDescription, introduced: 5.4)
     case c18
 
-    /// Working Draft for ISO C2x.
+    /// The identifier for the C2x draft language standard.
     @available(_PackageDescription, introduced: 5.4)
     case c2x
 
-    /// ISO C 1990 with GNU extensions.
+    /// The identifier for the GNU89 language standard.
     case gnu89
 
-    /// ISO C 1990 with GNU extensions.
+    /// The identifier for the GNU90 language standard.
     case gnu90
 
-    /// ISO C 1999 with GNU extensions.
+    /// The identifier for the GNU99 language standard.
     case gnu99
 
-    /// ISO C 2011 with GNU extensions.
+    /// The identifier for the GNU11 language standard.
     case gnu11
 
-    /// ISO C 2017 with GNU extensions.
+    /// The identifier for the GNU17 language standard.
     @available(_PackageDescription, introduced: 5.4)
     case gnu17
 
-    /// ISO C 2017 with GNU extensions.
+    /// The identifier for the GNU18 language standard.
     @available(_PackageDescription, introduced: 5.4)
     case gnu18
 
-    /// Working Draft for ISO C2x with GNU extensions.
+    /// The identifier for the GNU2x draft language standard.
     @available(_PackageDescription, introduced: 5.4)
     case gnu2x
 
-    /// ISO C 1990.
+    /// The identifier for the ISO9899-1990 language standard.
     case iso9899_1990 = "iso9899:1990"
 
-    /// ISO C 1990 with amendment 1.
+    /// The identifier for the ISO9899-199409 language standard.
     case iso9899_199409 = "iso9899:199409"
 
-    /// ISO C 1999.
+    /// The identifier for the ISO9899-1999 language standard.
     case iso9899_1999 = "iso9899:1999"
 
-    /// ISO C 2011.
+    /// The identifier for the ISO9899-2011 language standard.
     case iso9899_2011 = "iso9899:2011"
 
-    /// ISO C 2017.
+    /// The identifier for the ISO9899-2017 language standard.
     @available(_PackageDescription, introduced: 5.4)
     case iso9899_2017 = "iso9899:2017"
 
-    /// ISO C 2017.
+    /// The identifier for the ISO9899-2018 language standard.
```

Comment:
As pointed out by @abertelrud in #4247, these shouldn't be changed as proposed. The documentation published on apple.com is outdated one, so it shouldn't be the basis for changing the current documentation.

Reply:
Agreed. I will use the original designations.